src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -737,8 +742,8 @@ internal CookieCollection InternalGetCookies(Uri uri)
             int port = uri.Port;
             CookieCollection cookies = null;
 
-            List<string> domainAttributeMatchAnyCookieVariant = new List<string>();
-            List<string> domainAttributeMatchOnlyCookieVariantPlain = null;
+            var domainAttributeMatchAnyCookieVariant = new Collections.Generic.List<string>();
```

Comment:
The compiler couldn't distinguish between SortedList generic and non-generic. I will add the fully namespace here.